docker-compose.appstore-tycho.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
```

Comment:
Will this be needed after the appstore update?

Reply:
This doesn't have to here necessarily but thought might be useful. You are referring to the docker-compose file, right?

Reply:
The compose file, and in general all the tycho components. Ideally this project just uses the latest appstore develop image. The reason it can't in this scenario is an ordering problem right? Because Tycho hasn't been updated in the appstore requirements file yet and the `workspace` pr there hasn't merged so this PR for `helx-ui` needs the added tycho custom here?